### ISSUE TITLE
api scope tutorial: Error when adding spfx-api-scopes-tutorial.sppkg to appcatalog

### ISSUE THREAD

**Issue by EA12** [pnp/sp-dev-fx-webparts#468]
## Category

- [ ] Question
- [x] Bug
- [ ] Enhancement

## Expected or Desired Behavior

The Solution should be installed without any error in Field "App Package Error Message" 

## Observed Behavior

The Solution is added to the appcatalog with an empty ID, empty Name and with "Error in Application" in field "App Package Error Message"

## Steps to Reproduce

clone the git repository mentionend in [Consume the Microsoft Graph in the SharePoint Framework](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/use-aad-tutorial)

The git Project: https://github.com/SharePoint/sp-dev-fx-webparts/tree/master/tutorials/api-scopes
Open the package-solution.json and change the resource (under webApiPermissionRequests) from "Microsoft Graph" to "Microsoft.Azure.AgregatorService"

Build, bundle and package like mentioned in the tutorial.
Add the solution to the appcatalog

Our Tenant is "Defaultrelease" / all

When I deploy to the Workbench of my Tenant, I can add the WebPart.
The WebPart is then not working because of a [known issue](https://github.com/SharePoint/sp-dev-docs/issues/1406) but that is another Story.

Is the Problem the Release-Mode of my Tenant?

**Comment by EA12** [comment]
Further Investigation:
I created a HelloWorldWebPart with yo and added the graphclient by Import:
`import { MSGraphClient } from '@microsoft/sp-client-preview';`
I used the Client in the HelloWorldWebPart:
 ```
const client: MSGraphClient = this.context.serviceScope.consume(MSGraphClient.serviceKey);
client
      .api('/me')
      .get((error, response: any, rawResponse?: any) => {
        if (error) {
          console.error(error);
....

```
The solution can be deplyoed successfully to the same Tenant.
When I add the WebPart to a Page I get following error:
The requested operation is part of an experimental feature that is not supported in the current environment.

This might be related to my initial issue and it looks like it is all about Release-Mode?
